Sources/MixpanelInstance.swift

```diff
@@ -119,6 +119,19 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate, AEDele
         }
     }
     
+    /// The flushBatchSize property controls the number of events sent in a single network request to the Mixpanel server.
+    /// Batch size affects how data is flushed from the client to Mixpanel. By configuring the size of the batch, you can
+    /// optimize network usage and control how frequently the client communicates with the server.
```

Comment:
`/track` has a maximum batch size of 50: https://developer.mixpanel.com/reference/track-event#limits

Reply:
good catch!